EmployeeService.Tests/EmployeeServicesTest.cs

```diff
@@ -1,36 +1,89 @@
 using System;
 using Xunit;
-using EmployeeService.Tests;
-using Moq;
 using EmployeeServices.Model;
-using EmployeeServices.Services;
 using EmployeeServices.Controllers;
 using System.Collections.Generic;
 using Microsoft.AspNetCore.Mvc;
+using Moq;
+using EmployeeServices.Services;
+using AutoFixture;
+using AutoFixture.AutoMoq;
+using AutoFixture.Xunit2;
 
 namespace EmployeeService.Tests
 {
     public class EmployeeServicesTest
     {
         private readonly EmployeeServicesController _employeeController;
-        
+        private Mock<IEmployeeService> _mockEmployeesList;
+
+        public EmployeeServicesTest()
+        {
+            _mockEmployeesList = new Mock<IEmployeeService>();
+            _employeeController = new EmployeeServicesController(_mockEmployeesList.Object);
+        }
+
+        [Fact]
+        public void GetTest_ReturnsListofEmployees()
+        {
+           
+            var result = _employeeController.Get();
+            Assert.IsAssignableFrom<ActionResult<List<Employee>>>(result);
```

Comment:
need something more than just assignable. 
what is result? null? empty or an expected